snapd-glib/requests/snapd-get-notices.c

```diff
@@ -103,7 +103,7 @@ generate_get_snap_request (SnapdRequest *request, GBytes **body)
     if (self->timeout != 0) {
         add_uri_parameter_base (query, "timeout");
         // timeout in microseconds
-        g_string_append_printf (query, "%luus", self->timeout);
+        g_string_append_printf (query, "%lluus", (unsigned long long int)self->timeout);
```

Comment:
You should be able to do this: `"%" G_GINT64_FORMAT, self->timeout`.

Reply:
Should we use the approach from https://gitlab.gnome.org/GNOME/gnome-calendar/-/merge_requests/492 ?

Reply:
In this case timeout is a GTimeSpan, so it's never been explicitly chosen as an integer.

Reply:
Oh, clever!!! Thanks for the tip!

Reply:
Done!